demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -2920,6 +2924,11 @@ export class DemoMeetingApp
       }
     }
 
+    if (value === 'No Audio') {
+      // An empty media stream destination without any source connected to it
```

Comment:
Whats the purpose of this, e.g. if a builder were to ask? That should probably go into the comment.

Reply:
It's primarily for testing the `sendingAudioFailed` event, I don't see it being useful otherwise. Will add that as a comment

Reply:
Can maybe update the comment to say it's currently only used in the integration test for the new event?